map_data/state_regions/14_siberia.txt

```diff
@@ -14,7 +14,6 @@
         bg_coal_mining = 84
         bg_iron_mining = 96
         bg_lead_mining = 81
-        bg_sulfur_mining = 22
```

Comment:
This was added with a reason as balancing but also historically there is sulfur here.